test/externalTests/zeppelin.sh

```diff
@@ -106,6 +106,10 @@ function zeppelin_test
     sed -i "s|it(\('other accounts cannot pause'\)|it.skip(\1|g" test/token/ERC721/presets/ERC721PresetMinterPauserAutoId.test.js
     sed -i "s|it(\('other accounts cannot unpause'\)|it.skip(\1|g" test/token/ERC721/presets/ERC721PresetMinterPauserAutoId.test.js
     sed -i "s|it(\('prevents initialization'\)|it.skip(\1|g" test/proxy/utils/Initializable.test.js
+    #TODO: Add more cases here to fix new errors
+    sed -i "s|it(\('deposit'\)|it.skip(\1|g/3" test/token/ERC20/extensions/ERC20TokenizedVault.test.js
```

Comment:
You are almost there.
That `sed` command will substitute (`s`) every occurrence (`g`) that matches the given pattern. But it will do this for **each line**.  In this case, we can tell `sed` to view the whole file as a single line (`-z`) and then match and replace only the third occurrence:
```
sed -zi "s|it(\('deposit'\)|it.skip(\1|3" test/token/ERC20/extensions/ERC20TokenizedVault.test.js
```